packages/core/test/components/Divider.test.tsx

```diff
@@ -15,4 +15,10 @@ describe('<Divider />', () => {
     expect(wrapper.prop('bottom')).toBe(4);
     expect(wrapper.prop('top')).toBe(2);
   });
+
+  it('can pass top and bottom to spacing', () => {
```

Comment:
hrmm maybe a different test message here? `it('renders a short divider'`?

Reply:
Ack, meant to change this, thanks